bip78/src/receiver/mod.rs

```diff
@@ -113,3 +125,6 @@ pub struct NewOutputOptions {
     subtract_fees_from_this: bool,
 }
 
+pub fn create_uri(address: &Address, amount: &Amount, pj: &str) -> String {
+    format!("{}?amount={}&pj={}", address.to_qr_uri(), amount.as_btc(), pj)
```

Comment:
Are we going to add URL dep? for this and for #3 ?

Reply:
IDK, I would really, really like to move BIP21 to `bitcoin` and folks there are mostly averse to dependencies. I think encoding is not that hard to do manually if we have to.

Reply:
I thought we can have BIP21 stuff without URL but with Extras inside rust-bitcoin and then this payjoin separated repo leveraging BIP21 parsing but with a layer on top using also URL

Reply:
Encoding belongs to BIP21 because it's also [used for description](https://github.com/bitcoin/bips/blob/master/bip-0021.mediawiki#examples) etc. If we use a dependency there it should be [`percent-encoding`](https://crates.io/crates/percent-encoding) because `url` seems to have web semantics.

Reply:
are we sure we need to percent-encode pj?

it looks btcpay doesn't percent encode neither I found mention in the bip